devito/types/tensor.py

```diff
@@ -31,16 +67,22 @@ class TensorFunction(AbstractTensor, Differentiable):
         Staggering of each component, needs to have the size of the tensor. Defaults
         to the Dimensions.
     """
+    _is_TimeDependent = False
```

Comment:
blank line below, `_sub_type` logically belongs to another group of class attributes

Reply:
fixed